### ISSUE TITLE
Maintenance?

### ISSUE THREAD

**Issue by bzg** [ThomasFKJorna/emacs-docs#9]
Hi, we list org-mode documentation resources on [this page](https://orgmode.org/manuals.org).

I'd love to list https://emacsdocs.org/docs/emacs/The-Emacs-Editor on this page, but I wonder if this repository is still maintained.

Do you plan to sync it with latest Org documentation?

If you need help for this task, you can ask on the mailing list: `emacs-orgmode@gnu.org`.

Thanks!

**Comment by bzg** [comment]
Hello again, so it seems this documentation is not maintained - don't hesitate to ask for help to the Org community.

**Comment by bzg** [comment]
I'm closing this issue as there were no interest so far.

**Comment by dive** [comment]
I believe it is better to keep this issue open for other visitors... For some reason, https://emacsdocs.org appears higher in search results than https://www.gnu.org/software/emacs/manual, for example. And it could be misleading, because the documentation is a couple of versions behind, and there is no indication on specific pages which version is in question.

**Comment by bzg** [comment]
You're right: I hope the maintainers will reply one day!

**Comment by ns-rse** [comment]
Would love to see the pages generated by this site updated.

**Comment by tefkah** [comment]
Hi! Sorry for the lack of response. I do agree it would be good to update the documentation at some point, it's gone unaddressed for too long. I will commit to updating the docs somewhere in the next month, please ping (or email) me if that has not happened!

**Comment by bzg** [comment]
Thanks!

Perhaps you can explain how one can help with keeping the documentation up to date and call for volunteers?

I suspect you can find volunteers among Emacs and Org users.

**Comment by kickingvegas** [comment]
Would be great to see an update! Thanks much - this is such a nice repackaging of the docs.

**Comment by stfl** [comment]
> Hi! Sorry for the lack of response. I do agree it would be good to update the documentation at some point, it's gone unaddressed for too long. I will commit to updating the docs somewhere in the next month, please ping (or email) me if that has not happened!

Hi. I just stumbled across emacs-docss and was very excited to have a much more readable and searchable alternative to the default documentation. I would highly appreciate an update to the docs.
Thank you

**Comment by MooersLab** [comment]
Ditto for the calls for updating the docs!!

**Comment by Knusper** [comment]
Ping...

**Comment by bzg** [comment]
I'd say that giving up after three years would be fair enough. Let's wait a few more weeks ✌️

**Comment by jman-schief** [comment]
This project is (or was) a great try to provide a more modern interface to the excellent Emacs documentation not familiar with `info` or finding the official gnu.org website hard to parse.

However, being not maintained (the Emacs manual dates back to 27.2) and *still* being on top of search engine results provides nice-looking but outdated content, IMO a mixed value to users.

**Comment by Knusper** [comment]
Thr maintainer is still active on GitHub. He's just ignoring these messages
here 😞

**Comment by bzg** [comment]
I sent an email to @tefkah, hopefully this will get through.

**Comment by Knusper** [comment]
Any update?

**Comment by tusharhero** [comment]
If you are not going to maintain this project. Please take down
emacsdocs.org. This website seems to come up above gnu.org docs in
search results as well&hellip;

**Comment by AndersonTorres** [comment]
Can the site be donated to anyone interested to keep it??
